src/test/scala/org/labrad/TlsTest.scala

```diff
@@ -2,23 +2,23 @@ package org.labrad
 
 import java.io.File
 import org.labrad.data._
-import org.labrad.manager.TlsPolicy
+import org.labrad.manager.{ManagerUtils, TlsPolicy}
 import org.labrad.types._
 import org.scalatest.FunSuite
 import scala.concurrent.Await
 import scala.concurrent.duration._
 
 class TlsTest extends FunSuite {
 
-  import TestUtils._
+  import ManagerUtils._
 
   def connectWithMode(tls: TlsMode, includeCerts: Boolean = true)(implicit m: ManagerInfo): Unit = {
     val certs = if (includeCerts) {
       Map(m.host -> m.tlsHosts.certFiles.map(m.host))
     } else {
       Map.empty[String, File]
     }
-    withClient(m.host, m.port, m.credential, tls = tls, tlsCerts = certs) { c =>
+    withClient(m, tls = tls, tlsCerts = certs) { c =>
```

Comment:
The ManagerInfo does include its `TlsPolicy`, which determines how it handles incoming connections, but here we're configuring the client's `TlsMode` which is not the same thing. This determines what the client will do for an outbound connection; one could have different clients connect to the same manager with different modes, for example.

Similarly for certs, the manager info includes the manager's tls configuration, including which certs it uses for which hostnames, but here we're configuring the client's trusted root certificates.